### ISSUE TITLE
Missing documentation

### ISSUE THREAD

**Issue by marklagendijk** [ngrx/router-store#34]
I'm currently in the process of setting up a big project with Angular and ngrx. I managed to figure out most things, but am missing some information about the `@ngrx/router-store`:

- When should `@ngrx/router-store` be used? For every `@ngrx` app? Or only for certain use cases?
- Do existing mechanisms like `routerLink` and `router.navigate` still work with  `@ngrx/router-store`?
- How should you `@ngrx/router-store`? Should you always use actions for routing?
- How does `@ngrx/router-store`? work? If you can still use the old mechanisms, does that mean that they also fire the appropriate actions? Are custom changes to the route state (by composing the `routerReducer` with a custom reducer) automatically applied?

These are all the questions I could come up with. It would be nice if we could add some documentation about this to the README.

**Comment by jculverwell** [comment]
I found these two videos really helpful:

* https://angular-university.io/lesson/angular-ngrx-adding-the-angular-router-to-the-chat-application
* https://angular-university.io/lesson/angular-ngrx-ngrx-store-router-integration-time-travelling-with-the-dev-tools

**Comment by brandonroberts** [comment]
Agreed. I'll work on better docs unless someone wants to contribute a PR

**Comment by Hirse** [comment]
@jculverwell Unfortunately the first video seems to be behind a paywall.

@brandonroberts Could you provide some answers to the questions above?
